starter_code/queries.md

```diff
@@ -4,80 +4,72 @@
-
-<!-- Your Code Goes Here -->
+filter : { $and: [ {"ipo.valuation_amount": { $gte: "100000000" } }, { founded_year: { $lt: "2010" } } ] }
+project : { name: 1, ipo: 1}
 
 ### 5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
-
-<!-- Your Code Goes Here -->
+filter : { $and: [ { number_of_employees: { $lt: 1000 } }, { founded_year: {$lt: 2005} } ] }
+sort : { number_of_employees: -1}
 
 ### 6. All the companies that don't include the `partners` field.
-
-<!-- Your Code Goes Here -->
+filter : {partners: {$exists: false}}
 
 ### 7. All the companies that have a null type of value on the `category_code` field.
 
-<!-- Your Code Goes Here -->
+filter : { category_code: { $in: [ null ] } }
 
 ### 8. All the companies that have at least 100 employees but less than 1000. Retrieve only the `name` and `number of employees` fields.
-
-<!-- Your Code Goes Here -->
+filter : {$and: [ { number_of_employees: {$gte: 100} }, { number_of_employees: {$lt: 1000} } ] }
+project : { name: 1, number_of_employees: 1}
 
 ### 9. Order all the companies by their IPO price in a descending order.
-
-<!-- Your Code Goes Here -->
+sort : { "ipo.valuation_amount": 1}
 
 ### 10. Retrieve the 10 companies with more employees, order by the `number of employees`
 
-<!-- Your Code Goes Here -->
-
+sort : { number_of_employees: -1}
+limit : 10
 ### 11. All the companies founded on the second semester of the year. Limit your search to 1000 companies.
 
-<!-- Your Code Goes Here -->
-
-<!-- ### 12. All the companies that have been 'deadpooled' after the third year. -->
-
-<!-- Your Code Goes Here -->
-
+filter : {$and: [ { founded_month: {$gte: 7} }, { founded_month: {$lte: 12} } ] }
+limit : 1000
 ### 12. All the companies founded before 2000 that have an acquisition amount of more than 10.000.000
 
-<!-- Your Code Goes Here -->
+filter {$and: [ { founded_year: {$lt: 2000} }, { "acquisition.price_amount": {$gte: 10000000} } ] }
 
 ### 13. All the companies that have been acquired after 2010, order by the acquisition amount, and retrieve only their `name` and `acquisition` field.
-
-<!-- Your Code Goes Here -->
-
+filter : { "acquisitions.acquired_year": {$gt: 2010} }
+project : {name: 1, acquisition: 1}
+sort : {"acquisition.price_amount": -1}
 ### 14. Order the companies by their `founded year`, retrieving only their `name` and `founded year`.
-
-<!-- Your Code Goes Here -->
+project : {name: 1, founded_year: 1}
+sort : {founded_year: -1}
 
 ### 15. All the companies that have been founded on the first seven days of the month, including the seventh. Sort them by their `acquisition price` in a descending order. Limit the search to 10 documents.
-
-<!-- Your Code Goes Here -->
+filter : {$and: [ { founded_day: {$gte: 1} }, { founded_day: {$lte: 7} } ] }
+sort : {"acquisition.price_amount": -1}
```

Comment:
I don't think there's a day 0 or below.
You could simplify it by 
{ founded_day: {$lte: 7} }